src/System.Private.CoreLib/src/System/Text/DecoderNLS.cs

```diff
@@ -137,10 +137,10 @@ internal unsafe override int GetCharCount(byte* bytes, int count, bool flush)
 
             // Just call pointer version
             fixed (byte* pBytes = &bytes[0])
-                fixed (char* pChars = &chars[0])
-                    // Remember that charCount is # to decode, not size of array
-                    return GetChars(pBytes + byteIndex, byteCount,
-                                    pChars + charIndex, charCount, flush);
+            fixed (char* pChars = &chars[0])
```

Comment:
I would expect nested fixed statement to be formatted at least with proper indentation (ideally using '{}' because of more than one nested level here. Like here: https://msdn.microsoft.com/en-us/library/f58wzh21.aspx